mathesar/reflection.py

```diff
@@ -78,6 +78,7 @@ def reflect_columns_from_table(table):
         if not created and column.display_options:
             serializer = DisplayOptionsMappingSerializer(data=column.display_options,
                                                          context={DISPLAY_OPTIONS_SERIALIZER_MAPPING_KEY: str(column.plain_type)})
+            # If the type of column has changed, the display options won't match with existing options
```

Comment:
Can we not invalidate display options during reflection if we detect that they don't match the type?

Reply:
We don't store column type in a Django model, so there is no way to know if the column `type` has changed.

Reply:
We have access to the column type, we can check whether the stored display options make sense for that type. Is there a problem with that approach?

Reply:
That is what I am doing, I missed out `the valid display options won't match with existing options".